styles/layouts/_main-navigation.scss

```diff
@@ -53,6 +53,8 @@
 	font-family: 'Work Sans';
 	min-height: 72px;
 	width: 100vw;
+  border-bottom: 1px solid black;
```

Comment:
is this just in github or is this not formatted correctly?

Reply:
should not lint catch this before commit?

Reply:
husky hook ignore .css & .scss file